### PR TITLE
Add Translation Guideline

### PR DESCRIPTION
**Pull request by meowmeowmeowcat**
- Add Translation Guideline in `contribute.rst`

Demo: https://python-packaging-user-guide--948.org.readthedocs.build/contribute/#translations.

### CODE REVIEW

File: source/contribute.rst [hunk 1]
```diff
@@ -67,7 +68,49 @@ an agreed-upon interface for interoperability between packaging tools.
...
+
+If you are experiencing issues while you are working on translations,
+please open an issue on `Github`__.
+
+.. Note:: Any translations of this project should follow `reStructuredText syntax`__.
+
+.. __: https://weblate.org/
+.. __: https://hosted.weblate.org/projects/pypa/packaging-python-org/
+.. __: https://github.com/pypa/packaging.python.org/issues
+.. __: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
```
Comment (webknjaz) [inline source/contribute.rst:85]: Please, make the references named. It's hard to maintain things that are hard to map and easy to break links by accidentally injecting some reference in the middle.

File: source/contribute.rst [hunk 2]
```diff
@@ -67,7 +68,49 @@ an agreed-upon interface for interoperability between packaging tools.
...
+Translations
+============
 
+We use `Weblate`__ to manage translations of this project.
+Please visit the `packaging.python.org`__ project on Weblate to contribute.
+
+If you are experiencing issues while you are working on translations,
+please open an issue on `Github`__.
+
+.. Note:: Any translations of this project should follow `reStructuredText syntax`__.
```
Comment (webknjaz) [inline source/contribute.rst:80]: Normally, admonitions (and other RST directives) are lowercase.
```suggestion
.. note::

   Any translations of this project should follow `reStructuredText syntax`__.
```

File: source/contribute.rst [hunk 3]
```diff
@@ -67,7 +68,49 @@ an agreed-upon interface for interoperability between packaging tools.
...
+.. __: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
+
+Adding a language
+-----------------
+
+If your language is not listed on `Weblate`__, click the button
+"Start new translation" at the bottom of the languages list and add
+the language you want to translate.
+
+.. __: https://hosted.weblate.org/projects/pypa/packaging-python-org/
```
Comment (webknjaz) [inline source/contribute.rst:94]: This link could be reused.

File: source/contribute.rst [hunk 4]
```diff
@@ -67,7 +68,49 @@ an agreed-upon interface for interoperability between packaging tools.
...
+Example:
+
+Wrong: Translate the following text directly::
+
+  `some ref`_ -> `TRANSLATED TEXT HERE`_
+
+Right: Translate the following text with your own language and add reference to the 
+original language::
+
+  `some ref`_ -> `TRANSLATED TEXT HERE <some ref>`_
```
Comment (webknjaz) [inline source/contribute.rst:111]: Maybe we should follow the style of right/wrong from the bottom of this document?

**Comment by meowmeowmeowcat** [comment]
All suggestions were applied!

File: source/contribute.rst [hunk 5]
```diff
@@ -67,7 +68,55 @@ an agreed-upon interface for interoperability between packaging tools.
...
+  When translating the text in reference, please do not translate them directly.
+
+  | Wrong: Translate the following text directly:
+
+  .. code-block::
+
+      `some ref`_ -> `TRANSLATED TEXT HERE`_
+
+  | Right: Translate the following text with your own language and add reference to the 
+    original language:
```
Comment (webknjaz) [inline source/contribute.rst:113]: How about
```suggestion
  | Right: Translate the following text with your own language and add the original reference:
```

Comment (meowmeowmeowcat) [reply source/contribute.rst]: Sounds better.

File: source/contribute.rst [hunk 6]
```diff
@@ -67,7 +68,54 @@ an agreed-upon interface for interoperability between packaging tools.
...
+.. _Weblate: https://weblate.org/
+.. _packaging.python.org: https://hosted.weblate.org/projects/pypa/packaging-python-org/
+.. _Github: https://github.com/pypa/packaging.python.org/issues
+.. _reStructuredText syntax: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
+
+Adding a language
+-----------------
+
+If your language is not listed on `packaging.python.org`_, click the button
+"Start new translation" at the bottom of the languages list and add
```
Comment (webknjaz) [inline source/contribute.rst:93]:
```suggestion
:guilabel:`Start new translation` at the bottom of the language list and add
```

File: source/contribute.rst [hunk 7]
```diff
@@ -67,7 +68,54 @@ an agreed-upon interface for interoperability between packaging tools.
...
+Translations
+============
 
+We use `Weblate`_ to manage translations of this project.
+Please visit the `packaging.python.org`_ project on Weblate to contribute.
+
+If you are experiencing issues while you are working on translations,
+please open an issue on `Github`_.
+
+.. note::
```
Comment (webknjaz) [inline source/contribute.rst:80]: I guess it's more of a tip than a note:
```suggestion
.. tip::
```

File: source/contribute.rst [hunk 8]
```diff
@@ -67,7 +68,54 @@ an agreed-upon interface for interoperability between packaging tools.
...
+If you are not familiar with reStructuredText (RST) syntax, please read `this guide`_
+before translating on Weblate.
+
+**Do not translate the text in reference directly**
+
+  When translating the text in reference, please do not translate them directly.
+
+  | Wrong: Translate the following text directly:
+
+  .. code-block::
```
Comment (webknjaz) [inline source/contribute.rst:108]:
```suggestion
  .. code-block:: rst
```

File: source/contribute.rst [hunk 9]
```diff
@@ -67,7 +68,54 @@ an agreed-upon interface for interoperability between packaging tools.
...
+
+      `some ref`_ -> `TRANSLATED TEXT HERE`_
+
+  | Right: Translate the following text with your own language and add the original reference:
+
+  .. code-block::
```
Comment (webknjaz) [inline source/contribute.rst:114]:
```suggestion
  .. code-block:: rst
```

**Review by webknjaz — APPROVED** [review]

**Comment by meowmeowmeowcat** [comment]
@webknjaz Thanks for review!

**Pull request merged**